apps/dotcms-ui/src/app/view/components/_common/dot-autocomplete-tags/dot-autocomplete-tags.component.ts

```diff
@@ -100,7 +101,7 @@ export class DotAutocompleteTagsComponent implements OnInit, ControlValueAccesso
      * @param * fn
```

Comment:
doc

Reply:
Is there.

Reply:
![image](https://user-images.githubusercontent.com/37185433/154733840-c5598434-60ac-45e4-a260-9169a72b0b06.png)